packages/shared/src/functions.ts

```diff
@@ -13,3 +13,6 @@ export const bigintMin = (...args: bigint[]): bigint =>
 
 export const bigintMax = (...args: bigint[]): bigint =>
   args.reduce((max, current) => (current > max ? current : max));
+
+export const calculateBpsAmountFromAmount = (bps: bigint, amount: bigint) =>
```

Comment:
we have something like this in `packages/swap/src/utils/fees.ts`, maybe we can merge them

Reply:
Good catch. I didn't see the existence of the other function.
Moved it to shared as well as the test for it.

Reply:
just realized that they are different functions. Looking into how i can merge them